client/src/components/FileCard.jsx

```diff
@@ -0,0 +1,121 @@
+import './FileCard.css'
+
+import { Folder, DocumentTxt, DocumentImage, Play } from 'grommet-icons'
+
+const FileCard = ({ name, path, type, size }) => {
+  const splittedNameArray = name.split('.')
+  const fileExtension = splittedNameArray[splittedNameArray.length - 1]
+
+  const renderFileIcon = (fileExtension) => {
+    if (fileExtension === 'txt') {
+      return <DocumentTxt color='green' />
+    } else if (fileExtension === 'jpg') {
+      return <DocumentImage color='yellow' />
+    } else if (fileExtension === 'mp4') {
+      return <Play color='blue' />
+    } else {
+      return <DocumentTxt color='green' />

```

Comment:
this line duplicates the same thing at line 11
you can only use `if` without `else` here, coz condition you are checking always has only one parameter to check, so no need to have if-else.

```js
if (A) {
   // do "aaa"
}

if (B) {
   // do "bbb"
}

// default case is:
// do "xxx" 
```